cdc/sink/ddlsink/mysql/mysql_ddl_sink.go

```diff
@@ -137,11 +137,36 @@ func (m *DDLSink) execDDLWithMaxRetries(ctx context.Context, ddl *model.DDLEvent
 		retry.WithIsRetryableErr(errorutil.IsRetryableDDLError))
 }
 
+// isReorgOrPartitionDDL returns true if given ddl type is reorg ddl or
+// partition ddl.
+func isReorgOrPartitionDDL(t timodel.ActionType) bool {
+	// partition related ddl
+	return t == timodel.ActionAddTablePartition ||
+		t == timodel.ActionDropTablePartition ||
```

Comment:
Is it expected that these deletion operations  such as " Drop & truncate"complete quickly

Reply:
Change the implementation. Would not block for drop/truncate partition ddl.